big_tests/tests/metrics_helper.erl

```diff
@@ -29,8 +30,9 @@ get_counter_value(Host, Metric) ->
 assert_counter(Value, CounterName) ->
     assert_counter(ct:get_config({hosts, mim, domain}), Value, CounterName).
```

Comment:
Shouldn't it be the host type?

Reply:
It seems so, I will double check, but probably an oversight, good catch!